solver/src/settlement_simulation.rs

```diff
@@ -0,0 +1,72 @@
+use crate::encoding::EncodedSettlement;
```

Comment:
The code in this module has moved from settlement_submission.rs and been changed to simulate multiple settlements in batches.